themes/theme-gmd/pages/Filter/components/Content/index.jsx

```diff
@@ -191,13 +191,12 @@ class FilterContent extends PureComponent {
    */
   render() {
     const { filters } = this.props;
+    const right = <ApplyButton active={this.hasChanged} onClick={this.save} />;
 
     if (!filters) {
-      return null;
+      return <CloseBar title="titles.filter" right={right} />;
```

Comment:
Please move this to a getter at least since it's a duplicate declaration to below

Reply:
done